### PR TITLE
complete the task

### PR DESCRIPTION
**Pull request by barsuchellos**

### CODE REVIEW

**Review by ArtemHulkoDev — APPROVED** [review]
everything is fine)

File: src/inverseRobot.js [hunk 1]
```diff
@@ -24,7 +24,28 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const val = Object.values(robot);
+
+  for (let counter = 0; counter <= val.length; counter++) {
+    let c = 1;
+
+    for (; c < val.length; c++) {
```
Comment (Denchepornyuk) [inline src/inverseRobot.js:32]: you don't need two loops, you need to create a new empty object, iterate over the keys from the robot object and write the robot value as the key to the new object

File: src/inverseRobot.js [hunk 2]
```diff
@@ -24,7 +24,28 @@
...
+      if (val[counter] === val[c]) {
+        // eslint-disable-next-line no-param-reassign
+        robot = null;
```
Comment (Denchepornyuk) [inline src/inverseRobot.js:35]: nope, you can rewrite function arguments

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]

File: src/inverseRobot.js (hunk 1 above)
Comment (barsuchellos) [reply src/inverseRobot.js:32]: ok , I rewrite it)

File: src/inverseRobot.js [hunk 3]
```diff
@@ -24,7 +24,17 @@
...
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
+
+  for (const property in robot) {
+    if (obj.hasOwnProperty(robot[property])) {
+      return null;
+    }
+
+    obj[robot[`${property}`]] = property;
```
Comment (Denchepornyuk) [inline src/inverseRobot.js:34]: create variable for `robot[property]` and use a variable in two places

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]

**Review by denys-cheporniuk — APPROVED** [review]

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**